README.md

```diff
@@ -2,30 +2,77 @@
 
 An index of applications built for the nRF Connect SDK.
 
+----
+
 ## Contributing
 
-If you or your organization has a project that you'd like to include in the nRF Connect SDK App Index, you can open a pull request against this repository. See the details shown in the [`CONTRIBUTING`](./CONTRIBUTING.md) file for more information.
+If you or your organization has a project that you'd like to include in the nRF Connect SDK App Index, you can open a pull request against this repository. Read the [`CONTRIBUTING`](./CONTRIBUTING.md) file for more information.
+
+----
 
 ## Development
```

Comment:
At this point it would be good to mention that the index built in this section is using the default entries (from `index` directory). It is still good to build it to present the process, but the users will want to create their own index (to serve it in their local network, or wherever they want to).

Actually, they don't have to generate the `index.json`, or even build this site. They can just fill the file according to the schema, and serve it somewhere.

Reply:
Discussed with Filip on call. Actions:
- break the procedure into subsections
- move information about what is required out of contributing.md -- keep only info where to place the json if one wants to contribute to this repo
- include information about schema mandatory fields under Development